app/models/organisation.rb

```diff
@@ -4,6 +4,8 @@ class Organisation < ApplicationRecord
   has_many :forms
   has_many :users
 
+  has_many :admin_users, -> { where(role: :organisation_admin) }, class_name: "User"
```

Comment:
We wondered about whether this should be done in the Organisation or User model. We settled on putting it in the Organisation model as it felt more like canonical Rails. It also avoids adding a class method to the User model, which seemed nicer.

Reply:
I think this makes sense to me to have the method on the organisation model, my only question is whether `has_many` is the right tool for this job; would a simple method also do the job? As I understand it adding this relation means we also have methods for changing/adding to admin users, which I don't think we need?

Reply:
Good point, switching over to just use a method